README.md

```diff
@@ -3,35 +3,29 @@
 ![Push Tests](https://github.com/chanzuckerberg/single-cell-data-portal/workflows/Push%20Tests/badge.svg)
 [![codecov](https://codecov.io/gh/chanzuckerberg/single-cell-data-portal/branch/main/graph/badge.svg?token=iIXh8Rw0CH)](https://codecov.io/gh/chanzuckerberg/single-cell-data-portal)
 
-The Single Cell Data Portal enables the publication, discovery and exploration of interoperable single-cell datasets. Data contributors can upload, review and publish datasets for private or public use. Via the portal, data consumers are able to discover, download and connect data to visualization tools such as [cellxgene](https://chanzuckerberg.github.io/cellxgene/posts/cellxgene_cziscience_com) to perform further analysis. The goal of the Data Portal is to catalyze distributed collaboration of single-cell research by providing a large, well-labeled repository of interoperable datasets.
+The Single Cell Data Portal enables the publication, discovery and exploration of interoperable single-cell datasets. Data contributors can upload, review and publish datasets for private or public use. Via the portal, data consumers are able to discover, download and connect data to visualization tools such as [cellxgene Explorer](https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/README.md) to perform further analysis. The goal of the Data Portal is to catalyze distributed collaboration of single-cell research by providing a large, well-labeled repository of interoperable datasets.
 
-## Developers
+## Developer Guidelines
 
-### Pre-requisites
+### Pre-requisite installations and setups
 
 1. Install pre-commit: `pre-commit install` or check doc [here](https://pre-commit.com/)
-2. [Install and configure awscli](docs/awscli.md)
-3. [Configure ssh access](https://github.com/chanzuckerberg/single-cell-infra#ssh)
-4. [Install pyenv](https://github.com/pyenv/pyenv). Make sure you choose the most recent python version from `pyenv`. If `pyenv` is already installed on your machine, you might have to delete and reinstall it.
-5. Run `source venv/bin/activate` to start up a virtual environment.
+1. Set up you machine to be able to work with AWS using the instructions [here](https://czi.atlassian.net/wiki/spaces/DC/pages/332892073/Getting+started+with+AWS). Please ensure to follow the instructions for step 3 to enable AWS CLI access all the way to the bottom so that you are also set up for SSH access. When you run the final command that requires the team's infra repo, use `single-cell-infra`.
+1. Set up your Python virtual environment. You can choose to do this by [installing pyenv](https://github.com/pyenv/pyenv). Note that we only support Python versions up to and including 3.8. If `pyenv` is already installed on your machine, you might have to delete and reinstall it. You may also choose to set up your Python virtual environment through [`venv`](https://docs.python.org/3/library/venv.html) as an alternative.
```

Comment:
That's a good thought. Before updating the content here, I remembered that Dimple mentioned that she didn't know about virtual environments and I ended up writing a snippet about `venv` there. I'm actually a `venv` user -- I know there are supposedly fewer features but it's shipped with Python 3 and I've never run into any problems/limitations with it.

Reply:
Along these lines, I will delete the Python environment setup instructions here since I made notes about setting up `venv` in the Onboarding Doc [here](https://docs.google.com/document/d/1yUq8myuLpwBGfQjAXeAsvfWYXdoJuWJebI3QJdetQJQ/edit#heading=h.qjbpcgbyu518). Let me know if you disagree!